### DISCUSSION TITLE
Problem with sending commands.

### DISCUSSION THREAD

**Discussion by Zombie1995** [luc-github/ESP3D#601] [F.A.Q] [answered]
I have anycubic i3 mega printer and there's some problem with sending commands. Let's see commands A and B for example. If i send A one time, printer does it. For the second and more time I send it, printer doesn't do it. Then i send B one time, printer does it. For the second and more time I send it, printer doesn't do it. And if i send A again one time, printer does it and so on. What is the reason of this problem?

**Comment by luc-github** [comment]
what FW version are you using ?
what command typicaly are you talking about ?
from where are you sending these commands ? webUI ? Serial ? Telnet ? etc...

**Reply by Zombie1995** [reply]
Marlin 2, esp3d 2.1.1, commands from webUI; command 1(G1 Z10 F600), 2(G1 Z-10 F600)

**Comment by Legsmaniac** [comment]
Try G1 Z10 F600 then G1 Z0 F600

**Reply by Zombie1995** [reply]
Yeah, this is of course working.

**Reply by luc-github** [reply, marked as answer]
you misunderstood the gcode I think
you use real positions - when you should use relative positions
you cannot go to z-10, you need to jog from -10 mm (incremental change) from position 10mm

https://gcodetutor.com/gcode-tutorial/g90-g91-gcode.html

**Reply by Zombie1995** [reply]
Thank you very much again!!!
